robolectric/src/test/java/org/robolectric/TestRunners.java

```diff
@@ -2,56 +2,37 @@
 
 import org.junit.runners.model.InitializationError;
 import org.robolectric.annotation.Config;
-import org.robolectric.manifest.AndroidManifest;
 
 import java.util.Locale;
-import java.util.Properties;
 
 import static org.robolectric.util.TestUtil.resourceFile;
 
 public class TestRunners {
 
-  public static class WithDefaults extends RobolectricTestRunner {
-    public static final String SDK_TARGETED_BY_MANIFEST = "-v23";
-
-    public WithDefaults(Class<?> testClass) throws InitializationError {
+  public static class SelfTest extends RobolectricTestRunner {
+    public SelfTest(Class<?> testClass) throws InitializationError {
       super(testClass);
       Locale.setDefault(Locale.ENGLISH);
     }
 
     @Override
-    protected AndroidManifest getAppManifest(Config config) {
-      Properties properties = new Properties();
-      properties.put("manifest", resourceFile("TestAndroidManifest.xml").toString());
-      return super.getAppManifest(
-          new Config.Implementation(config, Config.Implementation.fromProperties(properties)));
+    protected Config buildGlobalConfig() {
+      return new Config.Builder(super.buildGlobalConfig())
+          .setManifest(resourceFile("TestAndroidManifest.xml").toString())
+          .build();
     }
   }
 
-  public static class MultiApiWithDefaults extends MultiApiRobolectricTestRunner {
-
-    public MultiApiWithDefaults(Class<?> testClass) throws Throwable {
+  public static class MultiApiSelfTest extends SelfTest {
```

Comment:
Could we just make this SelfTest then any of the existing usages of SelfTest would get the Multi Api behaviour? Any that don't work on all APIs e.g: an existing bug, we could control with an @Config(sdk = ) ?